extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/QuteProcessor.java

```diff
@@ -1431,6 +1436,90 @@ void validateTemplateInjectionPoints(TemplateFilePathsBuildItem filePaths, List<
         }
     }
 
+    @BuildStep
+    CustomTemplateLocatorLocationsBuildItem validateAndCollectCustomTemplateLocatorLocations(
+            CustomTemplateLocatorsBuildItem locationsBuildItem,
+            BuildProducer<ValidationErrorBuildItem> validationErrors) {
+
+        Collection<AnnotationInstance> locators = locationsBuildItem.getLocators();
+
+        // Collect duplicities
+        Map<String, String> duplicateLocationToLocator = locators
```

Comment:
Pls try to avoid streams, lambdas and method references if possible. We just recently removed a lot of occurences in https://github.com/quarkusio/quarkus/pull/25623 and https://github.com/quarkusio/quarkus/pull/25647 to make our build steps a bit faster.

Reply:
Sorry, I didn't know that. I refactored the code.